extras/ams_wrapper/src/api/models/vehicle_detection_adas_model.json

```diff
@@ -0,0 +1,3 @@
+{
+  "1.0":"car"
```

Comment:
Can we hold keys in the OV format? I guess label id from model output is either float or integer. 
Maybe it could be like?
`{ 1 : "car" }`
or
`{ 1.0 : "car" }`

Also, I belive https://github.com/openvinotoolkit/model_server/pull/208/files will provide labels for this model.

Reply:
Json load accepts only double quotes. It has to be "1.0"
We will merge with json model definitions afterwords.